website/content/api-docs/relatedtools.mdx

```diff
@@ -32,6 +32,8 @@ The following list of tools is maintained by the community of Vault users; Hashi
 - [Cruise Daytona](https://github.com/cruise-automation/daytona) - An alternative implementation of the Vault client CLI for services and containers. Its core features are the ability to automate authentication, fetching of secrets, and automated token renewal. Support for AWS, GCP, & Kubernetes Vault Auth Backends.
 - [Vault-CRD](https://vault.koudingspawn.de/) - Synchronize secrets stored in HashiCorp Vault to Kubernetes Secrets for better GitOps without secrets stored in git manifest files.
 - [vsh](https://github.com/fishi0x01/vsh) - Interactive shell with tab-completion. Allows recursive operations on paths. Allows migration of secrets between both KV versions.
+- [vault-cli](https://github.com/IBM/vault-cli) - A yaml based automation tool that bootstraps vault cluster(s) with the desired configuration (namespaces, endpoints, policies, roles, endpoint)
```

Comment:
Vault should be capitalized. Otherwise LGTM

Reply:
I fixed this for the submitter -- feel free to re-review!